File: Lab6-PHP_AJAX_JSON/php/recipe.php
<?php
header("Access-Control-Allow-Origin: *");

class Recipe implements JsonSerializable {
	private $id;
	private $author;
	private $name;
	private $type;	
	private $prep_time;
    private $servings;
    private $ingredients;
    private $method;

	public function __construct($id, $author, $name, $type, $prep_time, $servings, $ingredients, $method) {
		$this->id = $id;
        $this->author = $author;
        $this->name = $name;
        $this->type = $type;
        $this->prep_time = $prep_time;
        $this->servings = $servings;
        $this->ingredients = $ingredients;
        $this->method = $method;
	}

	public function jsonSerialize() {
        $vars = get_object_vars($this);
        return $vars;
    }
}

?>

File: Lab9-AspNet/AspMVCex/AspMVCex/Models/Recipe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AspMVCex.Models
{
    public class Recipe
    {
        public int Id { get; set; }

        public string author { get; set; }
        public string name { get; set; }
        public string type { get; set; }
        public string prep_time { get; set; }
        public int servings { get; set; }
        public string ingredients { get; set; }
        public string method { get; set; }

    }
}